docs/source/variables.rst

```diff
@@ -0,0 +1,1663 @@
+
+72- shortwave_radiation_surface
+===============================
+
+===================  ==================================
+name                 shortwave_radiation_surface
+description          Shortwave radiation at the surface
+unit                 W m-2
+variable_type        float
+axis                 ['space']
+populated_by_init    []
+populated_by_update  []
+required_by_init     []
+updated_by           []
+required_by_update   []
+===================  ==================================
+
+73- soil_absorption
+===================
+
+===================  =============================================
+name                 soil_absorption
+description          Shortwave radiation absorbed by topsoil layer
+unit                 W m-2
+variable_type        float
+axis                 ['space']
+populated_by_init    []
+populated_by_update  ['abiotic']
+required_by_init     []
+updated_by           ['abiotic']
+required_by_update   []
+===================  =============================================
+
+74- soil_c_pool_lmwc
+====================
+
+===================  =====================================
+name                 soil_c_pool_lmwc
+description          Soil low molecular weight carbon pool
+unit                 kg C m-3
+variable_type        float
+axis                 ['space']
+populated_by_init    []
+populated_by_update  []
+required_by_init     ['soil']
+updated_by           ['soil']
+required_by_update   ['soil']
+===================  =====================================
+
+75- soil_c_pool_maom
+====================
+
+===================  ===========================================
+name                 soil_c_pool_maom
+description          Soil mineral associated organic matter pool
+unit                 kg C m-3
+variable_type        float
+axis                 ['space']
+populated_by_init    []
+populated_by_update  []
+required_by_init     ['soil']
+updated_by           ['soil']
+required_by_update   ['soil']
+===================  ===========================================
+
+76- soil_c_pool_microbe
+=======================
+
+===================  ====================================
+name                 soil_c_pool_microbe
+description          Soil microbial biomass (carbon) pool
+unit                 kg C m-3
+variable_type        float
+axis                 ['space']
+populated_by_init    []
+populated_by_update  []
+required_by_init     ['soil']
+updated_by           ['soil']
+required_by_update   ['soil']
+===================  ====================================
+
+77- soil_c_pool_necromass
+=========================
+
+===================  ============================
+name                 soil_c_pool_necromass
+description          Necrotic organic matter pool
+unit                 kg C m-3
+variable_type        float
+axis                 ['space']
+populated_by_init    []
+populated_by_update  []
+required_by_init     []
+updated_by           ['soil']
+required_by_update   []
```

Comment:
This should have the same properties as the other `soil_c_pool_*` so should `required_by_init` and `required_by_update` should both be 'soil'

Reply:
This file is generated automatically when building the documentation - actually, we might want to git-ignore it to avoid confusion. The file to review is only the TOML one in the root directory.